corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -1489,6 +1495,88 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
         },
     });
 
+    /* Handle an individual menu item. Also contains a child list view */
+    const PersistentMenuItemView = Marionette.View.extend({
+        tagName: "li",
+        template: _.template($("#persistent-menu-item").html() || ""),
+        regions: {
+            tree: {
+                el: 'ul',
+                replaceElement: true
+            }
+        },
+        triggers: {
+            "click a": "click:persistent:menu:command"  // magically calls onClickPersistentMenuCommand
+        },
+        templateContext: function () {
+            const appId = formplayerUtils.currentUrlToObject().appId,
+                  imageUri = this.model.get('imageUri'),
+                  icons = {JUMP: 'fa-pencil', NEXT: 'fa-regular fa-folder'};
+            return {
+                imageUri: imageUri ? FormplayerFrontend.getChannel().request('resourceMap', imageUri, appId) : "",
+                iconClass: icons[this.model.get('navigationState')] || 'fa-list-ul',
```

Comment:
FYI we decided to explicitly make a navigationState for case list. It'll be `ENTITY_SELECT` https://github.com/dimagi/formplayer/pull/1603/commits/0f2f63947b0afb26976fe7fd800b7236103bd10f